README.md

```diff
@@ -1,11 +1,187 @@
+## About
+If you use your calendar often, then you must have a decent timeline of how you spend your time. Why not analyze it?
 
+Tip: If you fill in the gaps, your stats will be even more meaningful!
 
+Connect your Google Calendar and immediately see stats on how you've been spending your time.
 
 Mostly built for fun!
+
+
+## Features and Usage
+
+
+## Development
+
+Want to help contribute, fix a bug, or enhance an existing feature? Great! Here's how you can run Calendar Analytics on your local machine.
+
+### Installation
+
+#### Required Downloads
+
+Install the following, if you haven't already:
+- [Virtualenv](https://virtualenv.pypa.io/en/stable/) in a separate folder
+- [Django](https://www.djangoproject.com/). We use Python 2.7
+- [Sass](http://sass-lang.com/install) for front-end
+- [Bower](https://bower.io/)
+- Fork or clone this app at https://github.com/MaxFangX/calendar-analytics
+
+
+#### Setup
+
+- Turn on the Google Calendar API by following the following steps:
+	1. Use [this wizard](https://console.developers.google.com/start/api?id=calendar) to create or select a project in the Google Developers Console and automatically turn on the API. Click **Continue**, then **Go to credentials**.
+	2. On the **Add credentials to your project** page, click the **Cancel** button.
+	3. At the top of the page, select the **OAuth consent screen** tab. Select an **Email address**, enter a **Product name** if not already set, and click the Save button.
+	4. Select the **Credentials tab**, click the **Create credentials** button and select **OAuth client ID**.
+	5. Select the application type **Other**, enter the name "Calendar Analytics Local Project", and click the **Create** button.
+	6. Identify the client_id and client_secret.
+
+- If you're using bash, add the following as your environment variables in ~/.bashrc
+	```
+	export CJ_DJANGO_SECRET="still-secret"
+	export CJ_GOOGLE_CALENDAR_API_CLIENT_ID="YOUR_CLIENT_ID_HERE"
+	export CJ_GOOGLE_CALENDAR_API_CLIENT_SECRET="YOUR_CLIENT_SECRET_HERE"
+	export APP_ENVIRONMENT="dev"
+	```
+
+- Refresh your command line so that the secret keys are in your environment. You can also run `source ~/.bashrc`. You can check that it work with `echo $APP_ENVIRONMENT`
+
+- Cd into the project directory and run the following:
+	```
+	pip install -r requirements.txt
+	```
+
+- Run the following:
+	```
+	python manage.py bower install
+	```
+	Select "d3#^3.4.4 which resolved to 3.5.17 and is required by nvd3#1.8.5" for the right d3 version
+
+
+### Running the App
+Run the virtualenv by entering the virtualenv directory, and run:
+```
+source ENV-NAME/bin/./activate
+```
+
+Enter the calendar-analytics/cal directory
+
+Set up the database by running
+```
+python manage.py migrate
+```
+
+Run the app itself by running
+```
+python manage.py runserver
+```
+
+Finally, to live update the front-end when you make a change, run one of the following:
```

Comment:
"Finally, to generate `.css` files from your `.scss` changes automatically, run..."